deploy/databases/linux/mysql/roles/onafterstart/tasks/templates/mysql-config.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+integration_name: com.newrelic.mysql
+
+instances:
+  - name: mysql-server
+    command: status
+    arguments:
+        hostname: localhost
+        port: {{ database_port }}
+        username: newrelic
+        password: {{ database_password }}
+        # New users should leave this property as `true`, to identify the
+        # monitored entities as `remote`. Setting this property to `false` (the
+        # default value) is deprecated and will be removed soon, disallowing
+        # entities that are identified as `local`.
+        # Please check the documentation to get more information about local
+        # versus remote entities:
+        # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
+        remote_monitoring: true
+        # Value of '1' indicates the gathering of the metrics below are enabled. 
+        # They are disabled by default.
+        extended_metrics: 1
+        extended_innodb_metrics: 1
+        extended_myisam_metrics: 1
+    labels:
```

Comment:
I dont think those labels below are correct.
Instead we should set all the labels/tags that are defined and pass through from the deployer.

Reply:
added tags as labels within the play.